arches_component_lab/views/api/widgets.py

```diff
@@ -26,18 +27,32 @@ def update_i18n_properties(response):
 
 class WidgetDataView(View):
     def get(self, request, graph_slug, node_alias):
-        card_x_node_x_widget = models.CardXNodeXWidget.objects.filter(
-            node__graph__slug=graph_slug,
-            node__alias=node_alias,
-            node__source_identifier_id__isnull=True,
-        ).first()
+        card_x_node_x_widget = (
+            models.CardXNodeXWidget.objects.filter(
+                node__graph__slug=graph_slug,
+                node__alias=node_alias,
+                node__source_identifier_id__isnull=True,
+            )
+            .select_related("node")
+            .get()
+        )
 
         response = update_i18n_properties(
             JSONDeserializer().deserialize(
                 JSONSerializer().serialize(card_x_node_x_widget)
             )
         )
 
+        datatype = DataTypeFactory().get_instance(card_x_node_x_widget.node.datatype)
+        # When dropping support for v7.6, try/except can be removed
+        try:
```

Comment:
ugh. I can see this causing major headaches with widgets getting 2 different formats back from this API depending on which version they're on.

Reply:
Fair, but we're only intending to support 7.6 very briefly, and I'm aware of no plans to implement this for anything other than the reference datatype right now, which is 8+.